torch/_inductor/fx_passes/pre_grad.py

```diff
@@ -458,3 +493,69 @@ def transpose_matmul(A: torch.Tensor, B: torch.Tensor, Atrans: bool, Btrans: boo
     if Btrans:
         B = B.transpose(-1, -2)
     return torch.matmul(A, B)
+
+
+_pointwise_ops = [
+    operator.add,
+    operator.sub,
+    operator.mul,
+    operator.iadd,
+    operator.isub,
+    operator.imul,
+    operator.itruediv,
+    operator.ifloordiv,
+]
+
+
+@register_graph_pattern(
+    CallFunction(
+        operator.setitem,
+        Arg(),
+        Arg(),
+        CallFunction(
+            _pointwise_ops,
+            CallFunction(
+                operator.getitem,
+                Arg(),
+                Arg(),
+            ),
+            Arg(),
+        ),
+    ),
+    pass_dict=split_cat_pass,
```

Comment:
Why is the pass dict this?

Reply:
Typo, it should be ```dyn_rewritten_pass```. But putting it in ```split_cat_pass``` also works, so CI doesn't complain it.